_src/pattern-library/sass/components/_layouts.scss

```diff
@@ -0,0 +1,708 @@
+// ------------------------------
+// edX Pattern Library: Components - Layouts
+
+// About: Contains base settings and options for the preset default Pattern Library UI Layouts.
+// ----------------------------
+
+// #CONFIG
+// #GLOBALS
+// #UTILITY
+// #CANNED - IN VIEW
+// #CANNED - VIEW
+
+// ----------------------------
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$susy-flush:            layout($susy-default inside); // set susy to make columns flush
+$susy-reversed:         layout($susy-default $grid-direction-reversed); // set susy to make columns display in reverse order
+$susy-view:            layout($susy-default auto); // set susy to make columns view display in reverse order
```

Comment:
NOTE: differentiating layouts that would take up the entirety of the view/viewport with this naming convention prefix.